Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -507,6 +520,88 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::slice)
     return js_string(vm, string_part);
 }
 
+JS_DEFINE_NATIVE_FUNCTION(StringPrototype::split)
+{
+    auto string = ak_string_from(vm, global_object);
+    if (string.is_null())
+        return {};
+
+    auto* result = Array::create(global_object);
+    size_t result_len = 0;
+
+    size_t limit = NumericLimits<size_t>::max();
```

Comment:
IIRC `size_t` doesn't have a well defined size? Let's just hardcode 2^32 - 1 as the spec says, possibly `constexpr`'d (or just as 4294967295).